### ISSUE TITLE
`rocks albedos` broken

### ISSUE THREAD

**Issue by Siltala** [maxmahlke/rocks#14]
Hi,

Another quick report - the command `rocks albedos` currently fails for every asteroid (as far as I can tell) which has albedo information in SsODnet with rocks version 1.6.6.
This also used to work fine, so is another recent regression.

Example output:
```
$ rocks albedos Alinda  
Traceback (most recent call last):
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/core.py", line 839, in __init__
    super().__init__(**ssocard)  # type: ignore
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 4 validation errors for Rock
diamalbedo -> beaming -> 1
  value is not a valid float (type=type_error.float)
diamalbedo -> beaming -> 2
  value is not a valid float (type=type_error.float)
diamalbedo -> err_beaming -> 1
  value is not a valid float (type=type_error.float)
diamalbedo -> err_beaming -> 2
  value is not a valid float (type=type_error.float)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lauri/.local/bin/rocks", line 8, in <module>
    sys.exit(cli_rocks())
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1651, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1698, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 44, in get_command
    return echo()
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 340, in echo
    rock = core.Rock(id_, datacloud=catalogues, suppress_errors=not verbose)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/core.py", line 874, in __init__
    super().__init__(**ssocard)  # type: ignore
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 2 validation errors for Rock
diamalbedo -> beaming -> 1
  value is not a valid float (type=type_error.float)
diamalbedo -> err_beaming -> 1
  value is not a valid float (type=type_error.float)
  ```

**Comment by Siltala** [comment]
There may be more than one issue at involved. The error message for Ceres is different:
```
$ rocks albedos Ceres
Traceback (most recent call last):
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/core.py", line 839, in __init__
    super().__init__(**ssocard)  # type: ignore
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 30 validation errors for Rock
diamalbedo -> albedo -> 14
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 16
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 17
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 18
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 19
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 20
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 21
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 22
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 23
  value is not a valid float (type=type_error.float)
diamalbedo -> albedo -> 24
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 14
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 16
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 17
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 18
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 19
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 20
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 21
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 22
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 23
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_up -> 24
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 14
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 16
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 17
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 18
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 19
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 20
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 21
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 22
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 23
  value is not a valid float (type=type_error.float)
diamalbedo -> err_albedo_down -> 24
  value is not a valid float (type=type_error.float)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lauri/.local/bin/rocks", line 8, in <module>
    sys.exit(cli_rocks())
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1651, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1698, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 44, in get_command
    return echo()
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 340, in echo
    rock = core.Rock(id_, datacloud=catalogues, suppress_errors=not verbose)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/core.py", line 872, in __init__
    del offending_part[error["loc"][-1]]
IndexError: list assignment index out of range
```

**Comment by Siltala** [comment]
2005 UD has a different error still:

```
$ rocks albedos 2005 UD 
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3629, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 136, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 163, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 5198, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 5206, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'diameter'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lauri/.local/bin/rocks", line 8, in <module>
    sys.exit(cli_rocks())
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1651, in invoke
    cmd_name, cmd, args = self.resolve_command(ctx, args)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1698, in resolve_command
    cmd = self.get_command(ctx, cmd_name)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 44, in get_command
    return echo()
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/cli.py", line 350, in echo
    datacloud.pretty_print(rock, core.rgetattr(rock, param), param)
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/datacloud.py", line 100, in pretty_print
    *[str(catalogue[c][i]) if c else str(i + 1) for c in columns],
  File "/home/lauri/.local/lib/python3.10/site-packages/rocks/datacloud.py", line 100, in <listcomp>
    *[str(catalogue[c][i]) if c else str(i + 1) for c in columns],
  File "/usr/lib/python3.10/site-packages/pandas/core/frame.py", line 3505, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/usr/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3631, in get_loc
    raise KeyError(key) from err
KeyError: 'diameter'
```

**Comment by maxmahlke** [comment]
Fixed in version 1.6.7. Thanks for reporting!
